pom.xml

```diff
@@ -76,6 +76,7 @@
         <!-- Eliminates the file encoding warning. Of course, all of your files
             should probably be UTF-8 nowadays. -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <javaLanguage.version>1.7</javaLanguage.version>
```

Comment:
Above comment is from the previous commit. In this commit I add it back in with a 1.7 version.